sale_rental/models/product.py

```diff
@@ -20,7 +20,7 @@ class ProductProduct(models.Model):
         string='Related Rental Services')
 
     @api.one
-    @api.constrains('rented_product_id', 'must_have_dates', 'type', 'uom_id')
+    @api.constrains('rented_product_id')
```

Comment:
@aheficent runbot was failed for sure. But If you remove type constraint what will restrict user from creating/editing a stockable rent product? Rent product have to be service for sure. Beside field (1) must_have_dates, (2) type, and (3) uom_id is not readonly on product.product. These screenshot taken from runbot:

![sale_order_rental](https://cloud.githubusercontent.com/assets/1947376/23299986/323e04a6-fab6-11e6-844d-639345384229.png)

![sale_order_rental_1](https://cloud.githubusercontent.com/assets/1947376/23299994/39f4ae7a-fab6-11e6-913c-7c32ca190877.png)

I don't know what cause runbot failed (i tried to look into the log but i could not find it), but removing constraint will endanger the rental process